arches/app/templates/base-root.htm

```diff
@@ -45,6 +45,7 @@
 
     {% block css %}
         <link rel="stylesheet" href="{% webpack_static 'node_modules/font-awesome/css/font-awesome.min.css' %}" />
+        <link rel="stylesheet" href="{% webpack_static 'node_modules/primeicons/primeicons.css' %}" />
```

Comment:
Do we also want these in base.htm? I can imagine the controlled list manager migrating to prime icons.

Reply:
Yes, but not directly. `base.htm` defers css to `arches.scss`